fotoapparat/src/main/java/io/fotoapparat/hardware/v1/capabilities/CapabilitiesFactory.java

```diff
@@ -86,4 +88,52 @@ public Capabilities fromParameters(Camera.Parameters parameters) {
 		return result;
 	}
 
+	private DiscreetRange extractSupportsSensorSensitivityValue(Camera.Parameters parameters) {
+		final Set<Integer> isoValuesSet = new HashSet<>();
```

Comment:
@dmitry-zaitsev sometimes likes finals. :) For me if they are not there it's also making the code cleaner